packages/radio-group/test/lion-radio.test.js

```diff
@@ -1,11 +1,39 @@
-import { expect, fixture } from '@open-wc/testing';
+import { expect, fixture, html } from '@open-wc/testing';
 import '../lion-radio.js';
 
+/**
+ * @typedef {import('../src/LionRadio').LionRadio} LionRadio
+ */
+
 describe('<lion-radio>', () => {
   it('should have type = radio', async () => {
-    const el = await fixture(`
-      <lion-radio name="radio" value="male"></lion-radio>
+    const el = await fixture(html`
+      <lion-radio name="radio" .choiceValue="${'male'}"></lion-radio>
     `);
     expect(el.getAttribute('type')).to.equal('radio');
   });
+
+  it('can be reset when unchecked by defualt', async () => {
```

Comment:
Should these tests be moved to the ChoiceInputMixin suite? I think these tests apply for all single choice input fields right? The only specific thing is type="radio" so that should be the only test in this file I think